src/app/home-page/components/today-tasks/today-tasks.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { TaskService } from '../../../shared-module/services/task.service';
+import { Task } from '../../../shared-module/types/task';
+
+@Component({
+  selector: 'app-today-tasks',
+  templateUrl: './today-tasks.component.html',
+  styleUrls: ['./today-tasks.component.scss'],
+})
+export class TodayTasksComponent implements OnInit {
+  constructor(private taskService: TaskService) {}
+
+  taskList: Task[] = [];
+
+  ngOnInit() {
+    this.taskService
+      .getTodayTasks()
+      .subscribe((tasks) => (this.taskList = this.moveDoneTaskToEnd(tasks)));
```

Comment:
[just a note] there are hot and cold Observables. 
It's a good practice to unsubscribe from hot ones(like when data is constantly going outside of the Observable - think of BehaviorSubject or mouse click events), but not necessary to unsubscribe from cold(generated inside of Observable - you made a http call and it returned data). Here you do not necessarily need to unsubscribe but make sure to unsubscribe from Subjects, ticks, and dom events - to prevent memory leak